### PR TITLE
Deprecation: #95800 - Deprecate generating public URL for private ass…

### PR DESCRIPTION
**Pull request by linawolf**
…et files

refs https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-CoreApi/issues/1583

releases: master, 11.5

### CODE REVIEW

File: Documentation/ExtensionConfiguration/Index.rst [hunk 1]
```diff
@@ -62,15 +62,14 @@ Extension icon
...
-Choose or create an image of 64px width by 64px height.
+feature an icon using a svg file. This image should be located at
+:file:`Resources/Public/Icons/Extension.svg`.
 
 .. note::
 
-   Newer versions of TYPO3 support alternative formats and file locations.
-   However, for the sake of simplicity we will stick to the specification outlined
-   above.
+   PNG and GIF files are also allowed as Extensions icon. They are stored in
```
Comment (mbrodala) [inline Documentation/ExtensionConfiguration/Index.rst:70]:
```suggestion
   PNG and GIF files are also allowed as extension icon. They should be stored in
```

Comment (susannemoog) [reply Documentation/ExtensionConfiguration/Index.rst:70]: Shouldn't that be "[if used...], they have to be stored in"? "Should" suggests that I could store them somewhere else, which isn't the case as far as I know,

Comment (mbrodala) [reply Documentation/ExtensionConfiguration/Index.rst:70]: Indeed, extension icons *must* be in `Public`, otherwise the simply won't work.

Comment (linawolf) [reply Documentation/ExtensionConfiguration/Index.rst:70]: If you use `helhum/typo3-secure-web` that is true. But for the time being they also work if they are in the original location. However I prefere to have them in Public, too....

**Review by susannemoog — APPROVED** [review]

**Pull request merged**